pylettize/palettes/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+Default pylettize palettes.
+
+The following default palettes are available:
+- gruvbox
+- obama
+- primaries
+
+These are nothing more than text files with RGB hex values, making
+it quite easy to include more palettes to the default library in the future.
+"""
+
+OPTIONS = ["gruvbox", "obama", "primaries"]
```

Comment:
Maybe this doesn't belong here now. My idea was to maybe move logic for reading these default palettes into this module at some point, and that exposing the available options could be useful until then.